conf/gunicorn.conf.py

```diff
@@ -0,0 +1,2 @@
+bind = "unix:/tmp/gunicorn-web.sock"
```

Comment:
Via differs from our other apps in that Gunicorn is bound to a Unix socket rather than to an HTTP port. This is necessary because Via has an internal NGINX instance that [connects to Gunicorn on this socket](https://github.com/hypothesis/via/blob/7df9e122f094b1014af8df64386313322a630c68/conf/nginx/includes/app_upstream.conf#L12).

I don't think it's possible for our other apps that don't have an internal NGINX to use a Unix socket: I think Elastic Beanstalk's external NGINX instance needs to connect to Gunicorn over HTTP not a socket?

Also not sure whether it'd be desirable for internal NGINX's to connect to Gunicorn via HTTP: I think Unix sockets are more efficient.

So it's probably best to leave this as it is and let it differ between apps.